cmd/client-s3.go

```diff
@@ -833,10 +829,18 @@ func (c *s3Client) GetAccessRules() (map[string]string, *probe.Error) {
 		return map[string]string{}, probe.NewError(BucketNameEmpty{})
 	}
 	policies := map[string]string{}
-	policyRules, err := c.api.ListBucketPolicies(bucket, object)
-	if err != nil {
-		return nil, probe.NewError(err)
+	policyStr, e := c.api.GetBucketPolicy(bucket)
+	if e != nil {
+		return nil, probe.NewError(e)
+	}
+	if policyStr == "" {
+		return policies, nil
+	}
+	var p policy.BucketAccessPolicy
+	if e = json.Unmarshal([]byte(policyStr), &p); e != nil {
+		return nil, probe.NewError(e)
```

Comment:
@harshavardhana , the changes to GetAccessRules() and SetAccess() seems unconnected to this PR. Is this in error?

Reply:
It since you will have to update the minio-go anyways.

Reply:
ok,got it

Reply:
Fixed